docs/02_Mapping.md

```diff
@@ -59,3 +61,47 @@ Config
           entity_attribute: typesenseAddress
 ```
 
+# Auto embedding fields
+
+You can use the Typesense auto embedding fields to generate embeddings. 
+
+```yaml
+  embedding:
+    name: embedding
+    type: float[]
+    index: true
+    embed:
+      from: ["field_to_embed"]
+      model_config:
+        model_name: "..."
+        api_key: '...'
+        url: "..."
+```
+
+You can refer to the [Typesense](https://typesense.org/docs/27.1/api/vector-search.html#index-embeddings) documentation for the details of the configuration. 
+
+Here is an example for a local ollama embedding that would generate embeddings from the "summary" and "tags" fields, with the `nomic-embed-text` model.
+- Please note: `numDim` should be specified and corresponding to your model specifications
+- You need to prefix your model name with `openai/` if you are not using Typesense's own embedding engine.
+
+```
+  embedding:
+    name: embedding
+    type: float[]
+    index: true
+    mapped: false
+    numDim: 768
+    embed:
+      from: ["tags", "summary"]
+      model_config:
+        model_name: "openai/nomic-embed-text"
+        api_key: '<key>'
+        url: "http://localhost:11434/"
+```
+
+
+## Mapping
+
+Unless you want to store embeddings in your database, you can specify `mapped: false` in the configuration, the embeddings will only live in typesense.
+
+In this case, we recommend to exclude the fields from being retrieved when doing a query as.
```

Comment:
"In this case, we recommend excluding these fields from being retrieved in the query."